player/client.c

```diff
@@ -1373,11 +1373,10 @@ int mpv_unobserve_property(mpv_handle *ctx, uint64_t userdata)
 static void mark_property_changed(struct mpv_handle *client, int index)
 {
     struct observe_property *prop = client->properties[index];
-    if (!prop->changed && !prop->need_new_value) {
-        prop->changed = true;
-        prop->need_new_value = prop->format != 0;
-        client->lowest_changed = MPMIN(client->lowest_changed, index);
-    }
+    prop->changed = true;
+    if (prop->format != 0)
+        prop->need_new_value = true;
```

Comment:
Any specific reason for the if, instead of what was there before? I think it makes effectively no difference.